castle/cms/profiles/default/registry/controlpanel.xml

```diff
@@ -118,4 +118,40 @@
     </value>
   </record>
 
+  <record name="castle.es_index"
```

Comment:
why are the castle.es_index and castle.es_index_enabled being re-added to the registry ?